ui/src/app/utils/access-utils.tsx

```diff
@@ -405,26 +405,31 @@ const withInvalidDateHandling = (date) => {
   }
 };
 
-export const computeRenewalDisplayDates = ({
-  completionEpochMillis,
-  expirationEpochMillis,
-  bypassEpochMillis,
-}: AccessModuleStatus) => {
+interface RenewalDisplayDates {
+  lastConfirmedDate: string;
+  nextReviewDate: string;
+  moduleStatus: AccessRenewalStatus;
+}
+export const computeRenewalDisplayDates = (
+  status: AccessModuleStatus
+): RenewalDisplayDates => {
+  const { completionEpochMillis, expirationEpochMillis, bypassEpochMillis } =
+    status || {};
```

Comment:
this was the fix: allows destructuring even if `status` is null/undefined

Reply:
👍🏻